notes/orca-cli-capabilities.md

```
@@ -0,0 +1,111 @@
+| `orca worktree rm` | 删除 worktree |
+| `orca worktree ps` | 编排状态总览 |
+
+### 终端
+
+| 命令 | 说明 |
+|------|------|
+| `orca terminal list` | 列出活跃终端 |
+| `orca terminal create --command "claude"` | 创建终端 tab |
+| `orca terminal read --cursor <n>` | 增量读取输出 |
+| `orca terminal send --text "..."` | 发送输入 (含 enter) |
+| `orca terminal wait --for exit|tui-idle` | 等待终端状态 |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)